src/Compilers/CSharp/Portable/Syntax/SyntaxExtensions.cs

```diff
@@ -377,6 +377,12 @@ internal static CSharpSyntaxNode GetContainingDeconstruction(this ExpressionSynt
             }
         }
 
+        internal static bool IsOutDeclaration(this DeclarationExpressionSyntax p)
+        {
+            return p.Parent?.Kind() == SyntaxKind.Argument
+                && ((ArgumentSyntax)p.Parent).RefOrOutKeyword.Kind() == SyntaxKind.OutKeyword;
+        }
```

Comment:
Consider implementing the existing method `IsOutVarDeclaration` using the new method `IsOutDeclaration`.
```
return p.Designation.Kind() == SyntaxKind.SingleVariableDesignation && p.IsOutDeclaration();
```